allennlp/tests/models/bert_for_classification_test.py

```diff
@@ -0,0 +1,86 @@
+# pylint: disable=abstract-method,no-self-use
+from typing import Dict
+
+from pytorch_pretrained_bert.modeling import BertConfig, BertModel
+from _pytest.monkeypatch import MonkeyPatch
+
+from allennlp.common.testing import ModelTestCase
+from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data.fields import TextField, LabelField
+from allennlp.data.instance import Instance
+from allennlp.data.token_indexers import TokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.modules.token_embedders.bert_token_embedder import PretrainedBertModel
+
+
+@DatasetReader.register("bert_classification_test")
+class TestReader(DatasetReader):
+    def __init__(self,
+                 lazy: bool = False,
+                 token_indexers: Dict[str, TokenIndexer] = None,
+                 tokenizer: WordTokenizer = None) -> None:
+        super().__init__(lazy)
+        self._token_indexers = token_indexers or {}
+        self.tokenizer = tokenizer or WordTokenizer()
+
+    def _read(self, file_path: str):
+        #            2   3    4   3     5     6   8      9    2   14   12
+        sentence1 = "the quickest quick brown fox jumped over the lazy dog"
+        tokens1 = self.tokenizer.tokenize(sentence1)
+        label1 = "positive"
```

Comment:
Btw, realistic use of this model should add a `[CLS]` token at the start of the token list (and `[SEP]` at the end), not sure to what extent a test like this should also serve as an example of proper usage.

Reply:
actually, the PretrainedBertIndexer adds [CLS] and [SEP] automatically

Reply:
see https://github.com/allenai/allennlp/blob/master/allennlp/tests/data/token_indexers/bert_indexer_test.py#L24

Reply:
Ah yes, I see now it has default start-tokens/end-tokens when instantiated from PretrainedBertIndexer.